docs/operator_subscriptions/subscriptions.md

```diff
@@ -0,0 +1,190 @@
+
+---
+
+## Chained Subscription Algorithms
+
+Chained subscriptions allow the placement of new rules to depend on the outcome or parameters of rules created in a previous step (the "parent"). The specific algorithm to use for this chaining is specified in the subscription rule.
+
+The two main chained subscription algorithms are:
+
+### 1. `associated_site`
+
+**Purpose:**  
+Selects an associated RSE based on the `associated_site` attribute from the parent rule's RSE.
+
+**How it works:**  
+- When a rule is created on an RSE that has the `associated_site` attribute, the algorithm will select one of the associated sites based on the given index (`associated_site_idx`).
+- This enables multi-step workflows where the child rule is coordinated with the parent’s site.
+
+**Usage Example:**  
+If a parent rule is on an RSE with `associated_site="T1_FR_CCIN2P3,T1_DE_KIT"`, and you want the child rule to go to the first associated site:
+```json
+{
+  "algorithm": "associated_site",
+  "associated_site_idx": 1
+}
+```
+
+---
+
+### 2. `exclude_site`
+
+**Purpose:**  
+Places the child rule on an RSE that does **not** match the site of the parent rule.
+
+**How it works:**  
+- The algorithm finds the `site` attribute of the parent rule's RSE.
+- Modifies the RSE expression to exclude this site, ensuring the child rule lands elsewhere.
+
+**Usage Example:**  
+Used when you need to guarantee data is not replicated to the same site in multiple workflow stages.
+
+---
+
+## Split Rule Option
+
+While not an "algorithm" per se, the `split_rule` filter option is often used to create one rule per matching RSE, useful for fine-grained placement and accounting. It can be combined with chained subscriptions for even more advanced workflows.
+
+**Purpose:**  
+- When `split_rule` is specified in a subscription filter, rules are created separately for each RSE matching the expression.
+- The number of rules created is equal to the number of RSEs matching the expression.
+- If `"copies": "*"` is specified, the number of copies is set to the number of matched RSEs.
+
+---
+
+## Summary Table
+
+| Algorithm         | Purpose                                             | Key Parameters            |
+|-------------------|-----------------------------------------------------|---------------------------|
+| associated_site   | Child rule on parent’s associated site              | associated_site_idx       |
+| exclude_site      | Child rule avoids parent’s site                     |                           |
+| split_rule        | Rule per matching RSE (not chained)                 | split_rule (filter)       |
+
+---
+
+## Further Reading
+
+- [Transmogrifier details and implementation](transmogrifier.md)
+- [Rucio Data Management Documentation](https://rucio.readthedocs.io/en/latest/)
```

Comment:
This is the old Rucio docs link, it just says "The Rucio documentation has been migrated to https://rucio.cern.ch/documentation."